client/login/wp-login/index.jsx

```diff
@@ -79,25 +85,31 @@ class Login extends React.Component {
 		}
 	}
 
-	footerContent() {
+	footerLinks() {
 		const {
 			magicLoginEnabled,
 			magicLoginView,
 			translate,
 		} = this.props;
-		let loginLink;
 
-		if ( magicLoginEnabled && ! magicLoginView ) {
-			loginLink = <a href="#" onClick={ this.props.onMagicLoginRequestClick }>{ translate( 'Email me a login link' ) }</a>;
+		const footerLinks = [];
+
+		if ( magicLoginEnabled ) {
+			if ( magicLoginView === REQUEST_FORM ) {
+				return <a href="#" onClick={ this.onClickEnterPasswordInstead }>{ translate( 'Enter a password instead' ) }</a>;
```

Comment:
Tried this -- the button does not have much contrast at all:
<img width="471" alt="screen shot 2017-04-11 at 3 16 36 pm" src="https://cloud.githubusercontent.com/assets/1587282/24926841/f6493bf8-1eca-11e7-9be7-62c59a80c2e1.png">

Perhaps some other styling or markup will make it stand out in the future.